### PR TITLE
Bring back Gitpod Self Hosted page as Gitpod Dedicated stronger alternative

### LINKED ISSUE #3439: re-create landing page for /self-hosted
see [mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/website?node-id=9325%3A20313&t=3p5d2IB2WlYyBaZe-1)

Reviewer @MichaelAring

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
fixes #3439

### [**`Live Preview`**](https://deploy-preview-3444--gitpod-kumquat.netlify.app/self-hosted)

Also, See this [_internal conversation_](https://gitpod.slack.com/archives/C01KGM9ETU6/p1676876027048219) 👀

---

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3444"></a>

### CODE REVIEW

File: src/routes/self-hosted.svelte [hunk 1]
```diff
@@ -30,44 +23,51 @@
...
+  <p class="text-lg sm:text-2xl mt-micro mb-x-small mx-auto max-w-4xl">
+    Run Gitpod in your own Cloud account with Gitpod Dedicated, our self-hosted
+    alternative great for large teams.
+  </p>
+  <LinkButton
+    href="/dedicated"
+    text="Request early access"
+    variant="primary"
+    size="large"
+    data-analytics={`{"context":"self-hosted"}`}
```
Comment (Siddhant-K-code) [inline src/routes/self-hosted.svelte:37]: @jakobhero is this the correct way to add the analytics for the CTAs?

File: src/lib/contents/home/features.ts [hunk 2]
```diff
@@ -158,3 +158,30 @@ export const secondaryFeatures: Card[] = [
...
+  title: "Maximize control",
+  paragraph:
+    "Gitpod Dedicated is a single-tenant installation that runs your Cloud accounts or ours.</br></br>Maximize control, minimize overhead and create frictionless developer experiences.",
+  previewComponent: Workspaces,
+};
+
+export const devsDontCareAboutMachines: Feature = {
+  title: "Dev environments that don't care about machines",
+  paragraph:
+    "Gitpod Dedicated is a single-tenant installation that runs your Cloud accounts or ours.</br></br>Maximize control, minimize overhead and create frictionless developer experiences.",
```
Comment (MichaelAring) [inline src/lib/contents/home/features.ts:177]:
```suggestion
    "With Gitpod, anyone on your team can spin up ready-to-code dev environments in seconds.</br></br>Platform teams use us to achieve quick onboarding, better collaboration, and maximized security.",
```

File: src/routes/self-hosted.svelte [hunk 3]
```diff
@@ -30,44 +23,51 @@
   }}
 />
 
-<PlansAndPricing {pricingPlans} />
+<Section class="text-center">
+  <h2 class="text-5xl font-bold">Gitpod self-hosted alternative</h2>
+  <p class="text-lg sm:text-2xl mt-micro mb-x-small mx-auto max-w-4xl">
+    Run Gitpod in your own Cloud account with Gitpod Dedicated, our self-hosted
+    alternative great for large teams.
```
Comment (MichaelAring) [inline src/routes/self-hosted.svelte:30]:
```suggestion
    alternative for large teams.
```

File: src/routes/self-hosted.svelte [hunk 4]
```diff
@@ -30,44 +23,51 @@
...
-      src="/images/support/monitor-dark.png"
-      alt="Monitor"
-      class="w-40 hidden dark:block transform scale-110"
-    />
-    <div>
-      <h2 class="h4">Get a demo</h2>
-      <p class="mb-micro">
-        And learn how your engineering organization can scale with Gitpod.
+    <h2 class="h3 !mb-micro md:!mb-x-small align-middle">
+      Get started with Gitpod Dedicated
```
Comment (MichaelAring) [inline src/routes/self-hosted.svelte:59]:
```suggestion
      Get started
```

File: src/routes/self-hosted.svelte [hunk 5]
```diff
@@ -30,44 +23,51 @@
...
+    </h2>
+    <div class="max-w-[460px] text-p-large align-middle">
+      <p class="">
+        Gitpod Dedicate is best for engineering teams of 50 - 500 developers.
```
Comment (MichaelAring) [inline src/routes/self-hosted.svelte:63]:
```suggestion
        Complete this form to join our limited availability waitlist. 
```

File: src/routes/self-hosted.svelte [hunk 6]
```diff
@@ -30,44 +23,51 @@
...
+        Get Early Access today.
```
Comment (MichaelAring) [inline src/routes/self-hosted.svelte:64]:
```suggestion
```

**Review by MichaelAring — DISMISSED** [review]

**Comment by MichaelAring** [comment]
Please check in with me before merging @Siddhant-K-code

File: src/routes/self-hosted.svelte [hunk 7]
```diff
@@ -1,23 +1,16 @@
...
-  import { pricingPlans } from "$lib/contents/self-hosted";
-  import ViewAllFeatures from "$lib/components/self-hosted/view-all-features.svelte";
-  import Card from "$lib/components/ui-library/card/card.svelte";
   import LinkButton from "$lib/components/ui-library/link-button";
   import Section from "$lib/components/section.svelte";
+  import Features from "$lib/components/features.svelte";
+  import Signup from "$lib/components/dedicated/signup.svelte";
 </script>
 
 <OpenGraph
```
Comment (Siddhant-K-code) [inline src/routes/self-hosted.svelte:16]: @MichaelAring We need to rephrase this one also

**Comment by MichaelAring** [comment]
_@Siddhant-K-code: as discussed, here is the copy for the 4 cards_


H: Run Gitpod in your cloud account
T: Gitpod Dedicated is an isolated, single-tenant installation managed by Gitpod.


**Private connections**
Set up private connections to your dev resources

**Region of choice**
Deploy your dedicated instance in your region of choice

**Low latency**
Get faster startup times with your private instance

**Fully compliant**
Comply with local data residency requirements

**Comment by MichaelAring** [comment]
_@Siddhant-K-code : As discussed, please add these two bullet points on the left side at the bottom of the page next to the form:_

- Dedicated is a fully isolated, private installation of Gitpod that meets complex compliance standards
- Gitpod teams fully manage your instance to ensure you're always working on the latest version

File: src/lib/contents/home/features.ts [hunk 8]
```diff
@@ -158,3 +158,30 @@ export const secondaryFeatures: Card[] = [
...
   },
 ];
+
+/**Dedicated Features
+ * Using on /self-hosted page
+ *
+ */
+
+export const maximizeControl: Feature = {
+  title: "Maximize control",
```
Comment (MichaelAring) [inline src/lib/contents/home/features.ts:168]:
```suggestion
  title: "Full speed and flexibility",
```

File: src/lib/contents/home/features.ts [hunk 9]
```diff
@@ -158,3 +158,30 @@ export const secondaryFeatures: Card[] = [
...
+  paragraph:
+    "Gitpod Dedicated is a single-tenant installation that runs your Cloud accounts or ours.</br></br>Maximize control, minimize overhead and create frictionless developer experiences.",
```
Comment (MichaelAring) [inline src/lib/contents/home/features.ts:170]:
```suggestion
    "The maintenance and operation of your single-tenant instance are fully managed by us.</br></br>Your team always works with the latest version of Gitpod.",
```

File: src/lib/contents/home/features.ts [hunk 10]
```diff
@@ -158,3 +158,30 @@ export const secondaryFeatures: Card[] = [
...
+  previewComponent: Workspaces,
+};
+
+export const devsDontCareAboutMachines: Feature = {
+  title: "Dev environments that don't care about machines",
+  paragraph:
+    "With Gitpod, anyone on your team can spin up ready-to-code dev environments in seconds.</br></br>Platform teams use us to achieve quick onboarding, better collaboration, and maximized security.",
```
Comment (MichaelAring) [inline src/lib/contents/home/features.ts:177]:
```suggestion
    "Meet complex compliance and deployment needs with an installation of Gitpod in your region of choice.</br></br>Increase data protection and isolation with private connections.",
```

File: src/lib/contents/home/features.ts [hunk 11]
```diff
@@ -158,3 +158,30 @@ export const secondaryFeatures: Card[] = [
...
+
+export const maximizeControl: Feature = {
+  title: "Maximize control",
+  paragraph:
+    "Gitpod Dedicated is a single-tenant installation that runs your Cloud accounts or ours.</br></br>Maximize control, minimize overhead and create frictionless developer experiences.",
+  previewComponent: Workspaces,
+};
+
+export const devsDontCareAboutMachines: Feature = {
+  title: "Dev environments that don't care about machines",
```
Comment (MichaelAring) [inline src/lib/contents/home/features.ts:175]:
```suggestion
  title: "Maximize compliance and control",
```

**Review by MichaelAring — DISMISSED** [review]
copy changes

**Review by MichaelAring — APPROVED** [review]
HOLD - need to make one more change

File: src/routes/self-hosted.svelte [hunk 12]
```diff
@@ -1,73 +1,97 @@
...
+  import { dedicatedCards } from "$lib/contents/enterprise";
+  import CardSmall from "$lib/components/card/card-small.svelte";
 </script>
 
 <OpenGraph
   data={{
     description:
-      "Deploy and operate Gitpod yourself. Install Gitpod Self-Hosted on GKE, EKS, AKS or Kubernetes. Made for teams. Free for up to 10 users.",
-    title: "Gitpod Self-Hosted - Run Gitpod on your infrastructure",
+      "Install a private, single-tenant instance of Gitpod on your cloud account. Gitpod Dedicated is our self-hosted alternative for large teams",
```
Comment (MichaelAring) [inline src/routes/self-hosted.svelte:22]:
```suggestion
      "Install a private, single-tenant instance of Gitpod on your cloud account. Gitpod Dedicated is our self-hosted alternative for large teams",
      title: "Gitpod self-hosted alternative"
```

**Pull request merged**